lib/components/user/monitored-trip/trip-summary-pane.tsx

```diff
@@ -1,61 +1,190 @@
-import { FormattedList, FormattedMessage } from 'react-intl'
+import { Bell, BellSlash, Calendar, Clock } from '@styled-icons/fa-regular'
+import { FormattedMessage } from 'react-intl'
 import React from 'react'
 
 import { dayFieldsToArray } from '../../../util/monitored-trip'
 import { MonitoredTripProps } from '../types'
-import FormattedDayOfWeek from '../../util/formatted-day-of-week'
-import Strong from '../../util/strong-text'
+import LocationIcon from '@opentripplanner/location-icon'
+import styled from 'styled-components'
 
-import TripSummary from './trip-summary'
+import { InlineLoading } from '../../narrative/loading'
+
+import MonitoredDays from './trip-monitored-days'
+import TripSummary from './trip-duration-summary'
 
 /**
  * Displays the summary information of a monitored trip.
  */
+
+const SavedTripBody = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 0 15px;
+
+  @media (max-width: 600px) {
+    flex-direction: column;
+  }
+`
+
+const LocationDetails = styled.div`
+  align-items: start;
+  border-right: 1px solid #ddd;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  justify-content: center;
+  padding: 20px;
+  width: 50%;
+
+  @media (max-width: 600px) {
+    align-items: center;
+    border-bottom: 1px solid #ddd;
+    border-right: 0;
+    border-top: 1px solid #ddd;
+    width: 100%;
+  }
+`
+const ItineraryDetails = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 20px;
+  width: 50%;
+
+  @media (max-width: 600px) {
+    width: 100%;
+  }
+`
+const TextWIcon = styled.div`
+  align-items: center;
+  display: flex;
+  gap: 7px;
+  justify-content: left;
+  width: 250px;
+
+  svg {
+    flex-shrink: 0;
+  }
+`
+
+const TripDetailWithIcon = styled(TextWIcon)`
+  gap: 12px;
+
+  svg {
+    width: 16px;
+  }
+
+  &:last-of-type {
+    align-items: flex-start;
+
+    svg {
+      margin-top: 2px;
+    }
+  }
+`
+
+const TripDetailsList = styled.ul`
+  align-items: start;
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  gap: 15px;
+  justify-content: center;
+  padding: 0;
+`
+
+const ToggleNotificationButton = styled.button`
```

Comment:
Should this be a bootstrap button with link style instead?

Reply:
Sure, but is there a reason to use Bootstrap over the html button?

Reply:
I guess with the updated styling it is not necessary!